test/unit/L2/l2Migrator.test.ts

```diff
@@ -988,8 +1078,8 @@ describe('L2Migrator', function() {
         // that even in that scenario the delegator pool is not called
         const slot = ethers.utils.solidityKeccak256(
             ['uint256', 'uint256'],
-            // delegatorPools mapping is at storage slot 8
-            [ethers.constants.AddressZero, '8'],
+            // delegatorPools mapping is at storage slot 9
```

Comment:
This changed because I inserted the `tokenAddr` storage var in the middle of existing storage vars. After deploying on L2 using a ManagerProxy we would never change the existing storage layout we would only append to it, but in this case since we were not using a ManagerProxy prior I chose to group the Controller contract addresses together.